.github/workflows/CI.yml

```diff
@@ -10,13 +10,17 @@ on:
   pull_request:
     branches: [ master ]
 
+  # Run this workflow every day
+  schedule:
+    - cron: '0 0 * * *'
+
```

Comment:
Why? Shouldn't it be triggered by updates?

Reply:
it should and will be triggered by updates.

but since we have `master` branch  version specified in `rebar.config` for escalus, that would be nice to have some regular check to ensure that escalus master version is not broken and still suitable for XMPP component development.  Note that `rebar3 upgrade` is one of the CI steps